src/Management/src/EndpointCore/SpringBootAdminClient/SpringBootAdminApplicationBuilderExtensions.cs

```diff
@@ -64,14 +64,24 @@ public static void RegisterWithSpringBootAdmin(this IApplicationBuilder builder,
         {
             logger?.LogInformation("Registering with Spring Boot Admin Server at {0}", options.Url);
 
-            var result = httpClient.PostAsJsonAsync($"{options.Url}/instances", app).GetAwaiter().GetResult();
-            if (result.IsSuccessStatusCode)
+            HttpResponseMessage result = null;
+            string exceptionMessage = null;
+            try
+            {
+                result = httpClient.PostAsJsonAsync($"{options.Url}/instances", app).GetAwaiter().GetResult();
+            }
+            catch (HttpRequestException ex)
+            {
+                exceptionMessage = ex.Message;
+            }
```

Comment:
Same comment as earlier applies here. Is `InvalidOperationException` intentionally left out?